src/js/table.js

```diff
@@ -12,7 +12,11 @@ import diagnostic_download from './diagnostic_download';
 
 'use strict';
 
-const tableStyle = 'border: 1px solid black;';
+const cellClass = 'azad_cellClass ';
```

Comment:
I'm gonna merge this to save another round trip.
There are three things I'm going to fix directly in the next commit:
1) I've been using variable_name for variables THIS_IS_A_CONST for shared const variables (I don't bother when the scope is tiny), and functionName for functions. I reckon tableStyle and friends should be caps and underscores.
This convention (I think) is pythonic - I don't know what js people use - it seems a lot more diverse than python.
2) the thing you didn't know what I meant was a var, that should have been a const
3) I will do a const for the tax string and share it across the different locations.